packages/edge-login-ui-react/src/components/ModalAccountCreated.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+import { sprintf } from 'sprintf-js'
```

Comment:
Remove 'sprintf' and use built-in string manipulation

Reply:
Keep `sprintf`.